Fix the Hello World exercise so it passes with 'make test' on CI

I do not know if this is failing locally, but on Travis CI we are getting the following failure:


```
===> Compiling exercises/hello-world/src/hello-world.lfe failed
/home/travis/build/exercism/lfe/exercises/hello-world/src/hello-world.lfe:9999: unbound function: #(from 1)
make: *** [test] Error 1
The command "make test" exited with 2.
```

--- a/exercises/hello-world/src/example.lfe
+++ b/exercises/hello-world/src/example.lfe
@@ -1,0 +1,5 @@
+(defmodule hello-world
+  (export (hello-world 0)))
+
+(defun hello-world ()
+  "Hello, world!")

--- a/exercises/hello-world/src/hello-world.lfe
+++ b/exercises/hello-world/src/hello-world.lfe
@@ -1,5 +1,5 @@
 (defmodule hello-world
-  (export (from 1)))
+  (export (hello-world 0)))
 
 (defun hello-world ()
   "Hello, world!")

--- a/exercises/hello-world/test/hello-world-tests.lfe
+++ b/exercises/hello-world/test/hello-world-tests.lfe
@@ -5,4 +5,4 @@
 (include-lib "ltest/include/ltest-macros.lfe")
 
 (deftest hello-world
-  (is-equal "Hello, world!" (hello-world:hello-world))
+  (is-equal "Hello, world!" (hello-world:hello-world)))

